docker-hub/deactivate-account.md

```diff
@@ -48,5 +48,7 @@ Before deactivating an organization, please complete the following:
 
 Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.
 
+4. Remove all other members, including those in the **Owners** team, within the organization.
```

Comment:
Should this step be above the note above, or below the one below? It feels like those belong together (but not familiar with the exact steps)

Reply:
@thaJeztah good point, thanks for the catch. Yes, it should be above. I moved it to step 3 and

```
Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.

> This cannot be undone! Be sure you've gathered all the data you need from your organization before deactivating it.
{: .warning 
```
remains below the check list.